frontend/src/router/router.ts

```diff
@@ -40,6 +40,11 @@ const routes: Array<RouteRecordRaw> = [
     component: EditorPage,
     meta: { isAuth: true },
   },
+  {
+    path: Route.EditDocument,
```

Comment:
Already implemented. Merge develop into your branch (git checkout develop; git pull origin develop; git checkout feature/auth-ui-and-documents-page; git merge develop;)

Reply:
Sorry, not develop, feature/ws-rooms